docs/pages/choose-an-edition/teleport-cloud/faq.mdx

```diff
@@ -279,8 +279,9 @@ certificate or use a custom domain name.
 
 ### Where does Teleport Enterprise Cloud run?
 
-Teleport Enterprise Cloud is deployed within AWS, using S3 and DynamoDB for storage with AWS-managed encryption keys. All data is currently hosted in us-west-2.
-Currently there is no option to select an alternative region for storage.
+Teleport Enterprise Cloud is deployed within AWS, using S3 and DynamoDB for storage with AWS-managed encryption keys.
```

Comment:
I wonder if we can simplify this. The question that is being asked is not "what specific services does Teleport Cloud use" - it's only asking where it runs.

I would say something like this:

```
### Where does Teleport Enterprise Cloud run?

Teleport Cloud runs on Amazon Web Services. We run proxies
in a variety of regions all over the world, and allow customers
to [select the region](link..) where data is stored. 
```

I'd also remove the note about encryption keys, because it's covered in the next topic.

Reply:
thanks Zac, please see update